# HG changeset patch
# User Rik <rik@octave.org>
# Date 1514502139 28800
#      Thu Dec 28 15:02:19 2017 -0800
# Node ID 0d196d840c02f6d74664428039172a73583638bf
# Parent  aab2355f3a77065b0ed4c0ff133224c781084cf0
maint: move ols, gls, lscov from statistics to linear-algebra directory.

* scripts/linear-algebra/gls.m, scripts/linear-algebra/lscov.m,
scripts/linear-algebra/ols.m: Moved from statistics/base directory.

* scripts/linear-algebra/module.mk, scripts/statistics/base/module.mk:
Update build system.

diff --git a/scripts/statistics/base/gls.m b/scripts/linear-algebra/gls.m
rename from scripts/statistics/base/gls.m
rename to scripts/linear-algebra/gls.m
diff --git a/scripts/statistics/base/lscov.m b/scripts/linear-algebra/lscov.m
rename from scripts/statistics/base/lscov.m
rename to scripts/linear-algebra/lscov.m
diff --git a/scripts/linear-algebra/module.mk b/scripts/linear-algebra/module.mk
--- a/scripts/linear-algebra/module.mk
+++ b/scripts/linear-algebra/module.mk
@@ -4,30 +4,33 @@ scripts_linear_algebra_FCN_FILES = \
   %reldir%/bandwidth.m \
   %reldir%/commutation_matrix.m \
   %reldir%/cond.m \
   %reldir%/condeig.m \
   %reldir%/condest.m \
   %reldir%/cross.m \
   %reldir%/duplication_matrix.m \
   %reldir%/expm.m \
+  %reldir%/gls.m \
   %reldir%/housh.m \
   %reldir%/isbanded.m \
   %reldir%/isdefinite.m \
   %reldir%/isdiag.m \
   %reldir%/ishermitian.m \
   %reldir%/issymmetric.m \
   %reldir%/istril.m \
   %reldir%/istriu.m \
   %reldir%/krylov.m \
   %reldir%/linsolve.m \
+  %reldir%/lscov.m \
   %reldir%/logm.m \
   %reldir%/normest.m \
   %reldir%/normest1.m \
   %reldir%/null.m \
+  %reldir%/ols.m \
   %reldir%/orth.m \
   %reldir%/planerot.m \
   %reldir%/qzhess.m \
   %reldir%/rank.m \
   %reldir%/rref.m \
   %reldir%/subspace.m \
   %reldir%/trace.m \
   %reldir%/vech.m \
diff --git a/scripts/statistics/base/ols.m b/scripts/linear-algebra/ols.m
rename from scripts/statistics/base/ols.m
rename to scripts/linear-algebra/ols.m
diff --git a/scripts/statistics/base/module.mk b/scripts/statistics/base/module.mk
--- a/scripts/statistics/base/module.mk
+++ b/scripts/statistics/base/module.mk
@@ -2,29 +2,26 @@ FCN_FILE_DIRS += scripts/statistics/base
 
 %canon_reldir%_FCN_FILES = \
   %reldir%/center.m \
   %reldir%/cloglog.m \
   %reldir%/corr.m \
   %reldir%/corrcoef.m \
   %reldir%/cov.m \
   %reldir%/crosstab.m \
-  %reldir%/gls.m \
   %reldir%/histc.m \
   %reldir%/iqr.m \
   %reldir%/kendall.m \
   %reldir%/kurtosis.m \
   %reldir%/logit.m \
-  %reldir%/lscov.m \
   %reldir%/mean.m \
   %reldir%/meansq.m \
   %reldir%/median.m \
   %reldir%/mode.m \
   %reldir%/moment.m \
-  %reldir%/ols.m \
   %reldir%/ppplot.m \
   %reldir%/prctile.m \
   %reldir%/probit.m \
   %reldir%/qqplot.m \
   %reldir%/quantile.m \
   %reldir%/range.m \
   %reldir%/ranks.m \
   %reldir%/run_count.m \
